lib/engine-addon.js

```diff
@@ -734,6 +745,10 @@ module.exports = {
       @return {Object} Configuration object that will be provided to the engine.
     */
     options.engineConfig = function(env, baseConfig) {
```

Comment:
When lazyLoading is enabled, the first pass in this function `env` is set as `undefined`.